diffsims/tests/generators/test_simulation_generator.py

```diff
@@ -363,7 +362,7 @@ def test_calculate_diffraction2d_progressbar_single_phase(capsys):
     expected = "test phase: 100%|██████████| 10/10"  # also some more, but that is compute-time dependent
     # ignore possible flushing
     captured = captured.err.split("\r")[-1]
-    assert captured[: len(expected)] == expected
+    assert "████" in captured  # This can be faulty on CI, but we check for the bar
```

Comment:
These can be done with regex to allow any number of the block character. I'll give it a go later today

Reply:
https://github.com/CSSFrancis/diffsims/pull/3